server/graphql/mutations/expenses.js

```diff
@@ -97,6 +115,7 @@ export async function updateExpenseStatus(remoteUser, expenseId, status) {
   const res = await expense.update({ status, lastEditedById: remoteUser.id });
   if (status === statuses.APPROVED) {
     await _createActivity(expense, activities.COLLECTIVE_EXPENSE_APPROVED);
+    await _addUserIdToHostW9ReceivedList(expense);
```

Comment:
this doesn't work because any admin of the collective can approve the expense but they have no way to know if the host has received the W9.

Ideally, we would want the host to also approve expenses, but we don't support that at the moment.

So right now, the usual process is that one of the admins of the collective approves pending expenses then one of the admin of the host pays approved expenses. 

So I'd only run `_addUserIdToHostW9ReceivedList(expense);` when `status === statuses.PAID`

Reply:
Got it, thanks.

I would rather move both `_createActivity` and  `_addUserIdToHostW9ReceivedList ` to a hook on the expense model than creating one more `if` in this method. anyways, will do it.